datafusion/sql/src/expr/mod.rs

```diff
@@ -467,7 +467,7 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                 expr,
                 substring_from,
                 substring_for,
-                special: false,
+                ..
```

Comment:
```suggestion
                special: _
```

Maybe this so we can be informed of any new fields in Substring in the future?

Reply:
I think this is a very good practice, thank you @Jefffrey . Applied in [commit](https://github.com/apache/datafusion/pull/10242/commits/0d0fbe891cae0eac54407c1aa6930902b68a6575).